buildpack.md

```diff
@@ -612,12 +612,13 @@ In either case,
 
 The following additional environment variables MUST NOT be overridden by the lifecycle.
```

Comment:
This table appears to list env vars that are set on stack and should not be modified by the lifecycle. In this case `CNB_BUILDPACK_DIR` would **not** be set on the stack. The lifecycle should actively set it to the correct value for each buildpack. Because there is no existing case like this we may need to make a new section.

Reply:
yup, agree. i'll fix it.